### DISCUSSION TITLE
Unable to access webdav

### DISCUSSION THREAD

**Discussion by dteer** [photoprism/photoprism#4546] [Q&A]
The configuration has not changed and is deployed on the cloud server according to the documentation. It can be accessed on the web normally, and WEBDAV is in the enabled state. The log prompt for accessing WEBDAV is as follows:
photoprism-1  | time="2024-09-20T23:08:47Z" level=debug msg="server: OPTIONS /originals/ (401) [202.87µs]"

**Comment by srett** [comment]
[401 means unauthorized](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401).

Most likely you provided invalid credentials. In case you generated an access token, check under *Settings -> Account -> Apps and Devices* that it's not expired.

**Reply by dteer** [reply]
No credentials were generated, using the administrator's account and password directly

**Reply by graciousgrey** [reply]
Do you have 2fa enabled? In this case you need to generate an app password for webdav.

- https://docs.photoprism.app/user-guide/users/2fa/#step-3-app-passwords
- https://docs.photoprism.app/user-guide/sync/webdav/#credentials

More potential reasons why webdav could fail can be found in our troubleshooting checklist:
- https://docs.photoprism.app/getting-started/troubleshooting/#no-webdav-access
